src/lib.rs

```diff
@@ -25,6 +25,13 @@ lazy_static! {
     static ref ICON_FILE: CString = CString::new("icq").unwrap();
 }
 
+mod blist_node {
+    use lazy_static::lazy_static;
+    lazy_static! {
```

Comment:
Doesn't need to be a lazy_static, simply use `const FOO: &str = "foo"`

Reply:
done